en/firefox_privacy_notice.md

```diff
@@ -86,7 +86,7 @@ Read the full documentation [here](https://firefox-source-docs.mozilla.org/toolk
 
 ## If you use these features, Firefox will share data to provide you functionality and help us improve our products and services: {: #optional-features }
 
-### Firefox Suggest and Top Pick{: #searches }
+### Firefox Suggest and Top Pick {: #searches }
```

Comment:
This shouldn't have been left sitting there, it's showing up in production for all locales (including English) :-( 

<img width="727" alt="Schermata 2022-04-07 alle 14 20 19" src="https://user-images.githubusercontent.com/3644868/162197374-17913046-99f6-4b2a-8fe7-3aa550e22737.png">

Reply:
Fixed in https://github.com/mozilla/legal-docs/commit/861814aeb46b9ea2b37571c940f74f1b05808436